site/src/sphinx/server-annotated-service.rst

```diff
@@ -328,7 +345,7 @@ To catch an exception from every method in your service object, annotate your cl
         public HttpResponse hello() { ... }
     }
 
-To catch an exception from a single method in your service object, annotate your method with the annotation.
+To catch an exception from a single method in your service class, annotate your method with the annotation.
```

Comment:
How about:

    You can also annotate at method level to catch an exception from a single method in your service class.